pkg/crd/generator/generator.go

```diff
@@ -112,7 +109,7 @@ func (c *Generator) Do() error {
 		return fmt.Errorf("failed switching working dir: %v", err)
 	}
 
-	if err := b.AddDirRecursive("./pkg/apis"); err != nil {
+	if err := b.AddDirRecursive(fmt.Sprintf("./%s", c.APIsPath)); err != nil {
```

Comment:
If this is always relative, then we should update the help text for `api-path` commandline param to reflect that. 

I wish `filepath.Join(".", c.APIsPath)` worked for this case, but looks like doesn't.

nit: probably `"./" + c.APIsPath` reads better in this case.

Reply:
@droot updated